inductiva/utils/format_utils.py

```diff
@@ -237,61 +237,38 @@ def get_tabular_str(tabular_data: Union[Mapping[str, Iterable[Any]],
     return f"\n{table}\n"
 
 
-def currency_formatter(amount, currency="USD"):
-    """Format a currency amount into a human-readable string."""
-    # Define currency-specific information
-    currency_info = {
-        "USD": {
-            "symbol": "$",
-            "dollar_name": "dollar",
-            "cent_name": "cent"
-        },
-        "EUR": {
-            "symbol": "€",
-            "dollar_name": "euro",
-            "cent_name": "cent"
-        },
-        "GBP": {
-            "symbol": "£",
-            "dollar_name": "pound",
-            "cent_name": "penny"
-        },
-        # Add more currencies here if needed
-    }
-
-    currency_data = currency_info.get(currency, currency_info["USD"])
-
-    # If the amount is less than 0.01 (i.e., smaller than 1 cent)
-    if amount < 0.01:
-        return ("less than 0 cents for more details check "
+def currency_formatter(amount: float) -> str:
+    """Format a currency amount into a human-readable string.
+    Convert the amount to a string with a maximum of 10 decimal places.
+    If the amount is less than 0.0001 (i.e., smaller than 0.01 cents),
+    return a message indicating that the amount is less than 0.0001 USD.
+    If the amount is less than 0.1, show all decimal places until the
+    first two non-zero decimal values (e.g., 0.00012345 -> 0.00012).
+    TODO: Add support for other currencies. We need to get the currency from
+    the BE. For now, we are using USD.
+    """
+
+    currency_data = "US$"
+
+    # If the amount is less than 0.0001 (i.e., smaller than 0.01 cents)
+    if amount < 0.0001:
```

Comment:
This value should be stored in a global variable since it is used in multiple places.

Reply:
I'll update that in the original PR. (https://github.com/inductiva/inductiva/pull/1631)